diem-move/diem-transactional-test-harness/tests/diem_test_harness/block_metadata.exp

```diff
@@ -0,0 +1,5 @@
+processed 2 tasks
+
+task 1 'block'. lines 3-3:
+Events:
+ContractEvent { key: EventKey(11000000000000000000000000000000000000000a550c18), index: 0, type: Struct(StructTag { address: 00000000000000000000000000000001, module: Identifier("DiemBlock"), name: Identifier("NewBlockEvent"), type_params: [] }), event_data: "0000000000000000c5e34b925cf6875fec02d4b77adbd2d6006400000000000000" }
```

Comment:
The event output is rather difficult to read. Could you write a custom printer if it isn't too much work?

Reply:
Will do, but probably in a separate commit. (I have a bunch coming in the way so there will be chances for me to insert this soon.)